docs/4.1/admin-guide.md

```diff
@@ -1913,6 +1913,8 @@ When adding new local users you have to specify which Kubernetes groups they bel
 ```bsh
 $ tctl users add joe --k8s-groups="system:masters"
 ```
+!!! tip "NOTE":
+    In order to execute the above command, a Teleport role in RBAC needs to be created as `k8s-group`.
```

Comment:
It would be super handy if we had a reference role in github.

Reply:
Hey Ben, can you expand a little bit on this on what you mean? (like role definitions, or references mentioning type of role, and specific group in this case `k8s-group` for example)

Reply:
If we need to create a RBAC role, what does it look like? Is it the same as this one? https://gravitational.com/teleport/docs/kubernetes_ssh/#impersonation

Reply:
Yes, it's the `ClusterRole` and `ClusterRoleBinding` can be the same. You can configure the subject to be a Kubernetes `User` rather than a `ServiceAccount` if that's easier.

Reply:
Will link to example RBAC role in new PR for this.